src/modules/listItems.js

```diff
@@ -0,0 +1,90 @@
+import comments from './comment.js';
+import fetchData from './fetchAllData.js';
+import { getAPI, postAPI } from './involvementAPI.js';
+
+export const listLikes = () => {
+  getAPI().then((response) => {
+    const like = Array.from(document.querySelectorAll('.likes'));
+    for (let i = 0; i < like.length; i += 1) {
+      response.forEach((item) => {
+        if (item.item_id === like[i].id) {
+          if (like[i].childNodes.length <= 1) {
+            const small = document.createElement('small');
+            small.className = 'small';
+            small.textContent = `${item.likes} likes`;
+            like[i].appendChild(small);
+          } else {
+            like[i].removeChild(like[i].childNodes[1]);
+            const small = document.createElement('small');
+            small.className = 'small';
+            small.textContent = `${item.likes} likes`;
+            like[i].appendChild(small);
+          }
+        }
+      });
+    }
```

Comment:
- [ ] This code functionality is working correctly although for this project you are required to use the `async/await` syntax instead of using the `.then` syntax. please kindly make this change.
- [ ] [OPTIONAL] adding `Array.from` is not necessary in this case, because the result of `querySelectorAll` is already iterable.
- [ ] [OPTIONAL] try to name your variables in a way that semantically makes sense with what they are in this case `likes` makes more sense than `like` because this is an array of element**s** with the `.like` class.
- [ ] [OPTIONAL] try to use the new ES6 syntax looping mechanisms rather than an old-style for loop in this piece of code. It will make your code slightly easier to read and more concise, it can also avoid small bugs with using indexing.

```suggestion
    const likes = document.querySelectorAll('.likes');
    likes.forEach((like) => {
      response.forEach((item) => {
        if (item.item_id === like.id) {
          if (like.childNodes.length <= 1) {
            const small = document.createElement('small');
            small.className = 'small';
            small.textContent = `${item.likes} likes`;
            like.appendChild(small);
          } else {
            like.removeChild(like.childNodes[1]);
            const small = document.createElement('small');
            small.className = 'small';
            small.textContent = `${item.likes} likes`;
            like.appendChild(small);
          }
        }
      });
    });
```

Reply:
All changes applied and works fine, thanks for the nice suggestion